TIcTacToe/tic-tac-toe/src/components/Game.js

```diff
@@ -1,82 +1,71 @@
-import React, { Component } from 'react' 
+import React, { useState } from 'react' 
 import Board from './Board';
 import Utils from './Utils';
```

Comment:
it'll be better to import as a `calculateWinner`. Right now, it look like a module not a function. Functions should start with lower case and verb.